src/errors/types.rs

```diff
@@ -631,8 +631,23 @@ impl ErrorType {
             Self::MappingType { error } => render!(tmpl, error),
             Self::BytesTooShort { min_length } => to_string_render!(tmpl, min_length),
             Self::BytesTooLong { max_length } => to_string_render!(tmpl, max_length),
-            Self::ValueError { error } => render!(tmpl, error),
-            Self::AssertionError { error } => render!(tmpl, error),
+            Self::ValueError { error, .. } => {
+                let error = error.as_ref()
+                    .map(|v| Cow::Owned(v.as_ref(py).to_string()))
+                    .unwrap_or_else(Cow::Borrowed("None"));
```

Comment:
Aaah sorry looks like this should have been

```suggestion
                    .unwrap_or(Cow::Borrowed("None"));
```

Reply:
Yep figured that